src/backend/Comments.mo

```diff
@@ -225,4 +233,46 @@ module {
         );
         List.toArray(comments);
     };
+
+    
+    //check principal is admin or not
+    func isAdmin(p : Principal) :  Bool {
+        if(Principal.equal(p , Principal.fromText(ADMIN[0]))){
+            true
+        }else if(Principal.equal(p , Principal.fromText(ADMIN[1]))){
+            true
+        }else {
+            return false
+        }
+    
+    };
+
+    public func deleteComment(state: State, owner: Principal, commentHash: CommentHash) : DeleteResult {
+    // Check if user is an admin
+
+    if (not isAdmin(owner)) {
+        return #err(# NotAdmin)
```

Comment:
You may throw an Error here.

Explain why...

Reply:
https://m7sm4-2iaaa-aaaab-qabra-cai.icp0.io/?tag=2466699488

Reply:
because only admin can see and click delete button, otherwise something malicious happend